src/sage/crypto/key_exchange/diffie_hellman.py

```diff
@@ -0,0 +1,314 @@
+r"""
+Diffie-Hellman Public Key Exchange Scheme
+
+This module contains a toy implementation of the Diffie-Hellman public key exchange
+scheme.
```

Comment:
You should format your code to fit into 80 cols.
```suggestion
This module contains a toy implementation of the Diffie-Hellman public key
exchange scheme.
```